ironfish/src/network/messages/newBlockV2.test.ts

```diff
@@ -1,32 +1,55 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
-import { SerializedCompactBlock } from '../../primitives/block'
+import { BlockHeader, Target } from '../../primitives'
+import { CompactBlock } from '../../primitives/block'
+import { createNodeTest, useTxSpendsFixture } from '../../testUtilities'
 import { NewBlockV2Message } from './newBlockV2'
 
 describe('NewBlockV2Message', () => {
-  it('serializes the object into a buffer and deserializes to the original object', () => {
-    const compactBlock: SerializedCompactBlock = {
-      header: {
-        graffiti: Buffer.alloc(32, 'graffiti1', 'utf8').toString('hex'),
-        minersFee: '0',
-        noteCommitment: {
+  const nodeTest = createNodeTest()
+
+  function expectNewBlockV2MessageToMatch(a: NewBlockV2Message, b: NewBlockV2Message): void {
+    // Test transactions separately because Transaction is not a primitive type
+    expect(a.compactBlock.transactions.length).toEqual(b.compactBlock.transactions.length)
+    a.compactBlock.transactions.forEach((transactionA, transactionIndexA) => {
+      const transactionB = b.compactBlock.transactions[transactionIndexA]
+
+      expect(transactionA.index).toEqual(transactionB.index)
+      expect(transactionA.transaction.hash().equals(transactionB.transaction.hash())).toBe(true)
+    })
+
+    expect({ ...a.compactBlock, transactions: undefined }).toMatchObject({
+      ...b.compactBlock,
+      transactions: undefined,
+    })
+  }
+
+  // eslint-disable-next-line jest/expect-expect
```

Comment:
Same comment here re: jest expects

Reply:
Updating linting to allow for functions that start with expect

Reply:
Can you remove the `eslint-disable` here?